exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -864,6 +872,14 @@ public String getDigest() {
     return toString();
   }
 
+  public void setCacheFileRoot(String cacheFileRoot) {
+    this.cacheFileRoot = cacheFileRoot;
+  }
+
+  public void setBatchSize(long batchSize) {
+    this.recommendedBatchSize = batchSize;
```

Comment:
Would it be good to add an assert to check recommendedBatchSize <= PARQUET_READER_RECORD_BATCH_SIZE max_value?